JapaneseFilter/sections/specific.txt

```diff
@@ -46,6 +46,7 @@ yomogame252.com###text-81
 blog.with2.net##.ads-a
 sqool.net###text-67
 sqool.net##.stickyunit
+valorantchannel.livedoor.blog##.rss-blogroll
```

Comment:
I think it would be better blocked in the `AdGuard Annoyances` because of the blocking rule.